Extensions/probing.hh

```diff
@@ -281,15 +280,26 @@ inline double getReactivityScore(const Subsequence &leftBase, const bool isUnpai
 		    }
 		    infile.close();
 		}
-		if (probingData.size() < (leftBase.seq->n)) {
-			std::cerr << "Warning: chemical probing data file '" << getProbing_dataFilename() << "' misses " << (leftBase.seq->n - probingData.size()) << " data-row(s) " << std::endl << "         compared to the number of nucleotides in your input sequence." << std::endl << "         Missing values will be set to 0.0!" << std::endl;
+        
+        // needs reworking when changing the way the whitespace between two sequences is represented
+        unsigned int seqLength = 0;
+        std::vector<std::pair<const char*, unsigned> > inputSequences = getInputs();
+        if (inputSequences.size() == 2) {
+            seqLength = inputSequences.at(0).second + inputSequences.at(1).second + 1;
```

Comment:
should we rename `seqLength` into something that indicates the potential plural like `inputsLength` ?